### ISSUE TITLE
Kilt Installer Fails on S3 Bucket creation

### ISSUE THREAD

**Issue by jonahjon** [falcosecurity/kilt#15]
**Describe the bug**

Kilt MacOS Installer Fails on S3 Bucket creation

Command Used
`
./kilt-cfn-installer-macos -r us-west-2 MyMacroName ./falco.kilt.cfg
`

```
Getting AWS account and region...XXXXXXXX in us-west-2
Getting S3 bucket...Traceback (most recent call last):
  File "main.py", line 100, in <module>
  File "click/core.py", line 829, in __call__
  File "click/core.py", line 782, in main
  File "click/core.py", line 1066, in invoke
  File "click/core.py", line 610, in invoke
  File "main.py", line 54, in main
  File "main.py", line 90, in get_s3_bucket
  File "boto3/resources/factory.py", line 520, in do_action
  File "boto3/resources/action.py", line 83, in __call__
  File "botocore/client.py", line 357, in _api_call
  File "botocore/client.py", line 676, in _make_api_call
botocore.exceptions.ClientError: An error occurred (IllegalLocationConstraintException) when calling the CreateBucket operation: The unspecified location constraint is incompatible for the region specific endpoint this request was sent to.
[59854] Failed to execute script main
```

**Comment by admiral0** [comment]
Sorry for late answer - the installer itself has been rewritten recently for a better experience (pyinstaller was causing issues on old glibc's)

The issue seems to be conflict between aws s3 client config and provided region. This is hopefully fixed in the new version

**Comment by admiral0** [comment]
This has been fixed in v0.1.5
